js/components/slider.js

```diff
@@ -116,19 +123,16 @@ var Slider = createReactClass({
   },
 
   onMouseDown: function() {
-    this.dragging = true;
+    this.setState({
+      isDragging: true
+    });
   },
 
   onMouseUp: function(event) {
-    this.dragging = false;
```

Comment:
Do we want to set the state 'isDragging' to false here?

Reply:
Yes, I must've removed this by accident. Will fix.